ocs_ci/ocs/resources/storage_cluster.py

```
@@ -155,6 +157,149 @@ def verify_osd_tree_schema(ct_pod, deviceset_pvcs):
+        return
+
+    patch_body = json.dumps({"spec": {"storageClasses": updated_classes}})
+    try:
+        consumer_ocp.patch(
+            resource_name=consumer_name,
+            params=patch_body,
+            format_type="merge",
+        )
+        log.info(
+            f"Removed {sc_name} from StorageConsumer "
+            f"{consumer_name}.spec.storageClasses"
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)